pkg/descriptions.json

```diff
@@ -64,8 +64,13 @@
     "CommonTypes": []
   },
   {
-    "Name": "Microsoft.NETFramework.Compatibility",
-    "Description": "References a number of .NET Core libraries that are intended to provide support for more APIs that exist on .NET Framework.",
+    "Name": "Microsoft.Windows.Compatibility",
+    "Description": "Windows Compatibility Pack for .NET Core that includes libraries intended to provide support for more APIs that exist on .NET Framework.",
```

Comment:
> Windows Compatibility Pack for .NET Core that includes libraries intended to provide support for more APIs that exist on .NET Framework.

I'd say:

> This Windows Compatibility Pack provides access to APIs that were previously available only for .NET Framework. It can be used from both .NET Core as well as .NET Standard.